src/project/qml/MuseScore/Project/internal/ProjectPropertiesView.qml

```diff
@@ -38,34 +39,43 @@ StyledListView {
     property NavigationPanel navigationPanel: null
     property int navigationColumnStart: 0
 
+    scrollBarPolicy: ScrollBar.AlwaysOn
+
     model: projectPropertiesModel
 
-    delegate: PropertyItem {
-        width: root.width
+    delegate: RowLayout{
+        anchors.left: parent ? parent.left : undefined
+        anchors.right: parent ? parent.right : undefined
+        spacing: 0
+        PropertyItem {
+            width: root.width - 10
```

Comment:
What does this number mean? Also, you should use Layout.preferredWidth inside layouts